sdk/translation/azure-ai-translation-text/samples/README.md

```diff
@@ -48,8 +48,8 @@ The appropriate constructor is invoked in each sample to create a `TextTranslati
 <!-- SNIPPET: sample_text_translation_client.create_text_translation_client_with_credential -->
 
 ```python
```

Comment:
For the snippet updates in this file you can run this script to automatically fix it:

`~/azure-sdk-for-python/sdk/translation/azure-ai-translation-text$ python ../../../tools/azure-sdk-tools/ci_tools/snippet_update/python_snippet_updater.py .`

Reply:
It should be fixed now.